modules/client-configuration/pages/virtualization.adoc

```diff
@@ -78,294 +77,6 @@ You can add virtual guests on a pacemaker cluster within the {productname} {webu
 
 Virtual Guests managed by a cluster can be live migrated.
 
-////
```

Comment:
I'm not sure if this section should be removed. Is this one linked to caasp or virtualization in general?

Reply:
I thought it was virt in general but it is referring to pacemaker which is a CaaSP reference right?